docs/specs/parsing.md

```diff
@@ -13,18 +18,75 @@
     - `DateTime`, `DateTimeOffset`
```

Comment:
I would add BigInteger to the list. I think it might be required to be able to implement full JSON parsing support.